doc/src/contributing/new-contributors-guide/dev-setup.md

```diff
@@ -175,7 +175,7 @@ You may want to take advantage of using virtual environments to isolate your dev
 If you use `conda`, you can use it to create a virtual environment:
 
 ```bash
-$ conda create -n sympy-dev python=3 mpmath flake8
+$ conda create -n sympy-dev -c conda-forge hypothesis python=3 mpmath flake8 pytest
```

Comment:
In gh-25492 I decided to move these to `requirements-test.txt` because the number of test dependencies is growing. Does a requirements file also work with `conda`?

Reply:
@oscarbenjamin  That might be a bit tricky. Installing using pip is fine ( `pip install -r requirements-test.text`), but if you're trying to do use `conda`, you may have to do `conda install -c conda-forge --file requirements-test.txt` to ensure it installs `Hypothesis` correctly. My worry is, it'll try to install _all_ the packages via `conda-forge`

Reply:
requirements files also work with conda as long as the package names are the same, which they usually are. 

> My worry is, it'll try to install all the packages via conda-forge

That's actually what you want. It's generally good to use only conda-forge. Mixing conda-forge and defaults is generally bad. Even so, `-c conda-forge` only adds conda-forge as a channel, so it would still include `defaults` (this is more about the user's setting of the conda `channel_priority` setting).

Reply:
Let's change this (and the pip instructions) to use the requrements-dev file. Also I feel like the instructions for doing an editable install should be separate, or just removed (it isn't really necessary to do an editable install to develop SymPy).

Reply:
@asmeurer Since `dev-setup.md`, is solely for the user to install dependencies and get started, wouldn't the use of `requirements-test.txt`  bypass having to install via `setup.py` even for pip? That is:

Should we change `pip install -e ".[dev]"` to  `pip install -r requirements-test.txt` 

Also, conda would now become `conda create -n sympy-dev -c conda-forge --file requirements-test.txt
`?

Reply:
Yes, exactly.

Reply:
But also note that that file doesn't actually exist yet, until https://github.com/sympy/sympy/pull/25492 is merged.

Reply:
After gh-25495 the file was added as `requirements-test.txt` but @asmeurer suggested in gh-25492 that the file should be renamed to `requirements-dev.txt` and other things like `flake8` should be added.

Perhaps that change can be made in this PR.